storage/src/mmr/journaled.rs

```diff
@@ -492,6 +485,21 @@ impl<E: RStorage + Clock + Metrics, H: CHasher> Mmr<E, H> {
         Some(self.pruned_to_pos)
     }
 
+    /// Close the MMR, syncing any cached elements to disk and closing the journal.
+    pub async fn close(mut self, h: &mut impl Hasher<H>) -> Result<(), Error> {
+        self.sync(h).await?;
+        self.journal.close().await?;
+        self.metadata.close().await.map_err(Error::MetadataError)
+    }
+
+    /// Close and permanently remove any disk resources.
+    pub async fn destroy(self) -> Result<(), Error> {
+        self.journal.destroy().await?;
+        self.metadata.destroy().await?;
+
+        Ok(())
```

Comment:
At some point, would be nice to add a linter to enforce return is on a standalone line or isn't (for consistency).

Maybe that is a bit overkill but have noticed our code isn't consistent here.

(no change required, just calling it out)